package.json

```diff
@@ -27,15 +25,10 @@
   },
   "engines": {
     "cordovaDependencies": {
-      "2.0.0": {
```

Comment:
We could keep these version to see how it was used in the past if someone looks at the current package?

Reply:
I have never understand how the engines work, or are supposed to work, but as I understand it, it would install an older version of the plugin based on the installed platform version if necessary. So I think it should not be removed, but I’m not certainly sure.

but the version 7 engines should be updated as at the moment it will prevent the install on any CLI version since it has the  > 100

Reply:
Restored.